docs/marketplace/user/back-office-user-guides/202106.0/catalog/products/product-options/managing-product-options.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Managing product options
+last_updated: Apr 21, 2021
+description: Use this document to manage product options in the Back Office.
+template: back-office-user-guide-template
+---
+
+This document describes how to manage product options.
+
+## Prerequisites
+
+To start working with product options, go to **Catalog** > **Product Options**.
+
+
+Each section in this article contains reference information. Make sure to review it before you start, or just look up the necessary information as you go through the process.
+
+
+## Filtering product options by merchants
```

Comment:
This looks more like tips and tricks than a managing option of the product option. Consider using a different format for this information.